src/watch.ts

```diff
@@ -60,10 +60,10 @@ export class Watch {
             } else {
                 obj = JSON.parse(data) as WatchUpdate;
             }
-            if (obj.type && obj.object) {
+            if (typeof obj === 'object' && obj.object) {
```

Comment:
Not sure the `typeof` check achieves much - the `obj.object` check will surely catch any non-object cases?  Seems like the goal of this PR is simply to allow the check to pass even if `obj.type` is `undefined` so perhaps just replace this with `if (obj.object)` or `if (obj && obj.object)`?

Reply:
I just wanted to make sure the reply was an object (as it should be as far as I know) so I think this line is just a simple improvement!